share/backend_config.ini

```diff
@@ -21,10 +21,10 @@ database = zonemaster
 database_file = /var/lib/zonemaster/db.sqlite
 
 [ZONEMASTER]
-max_zonemaster_execution_time            = 300
+#max_zonemaster_execution_time           = 600
 number_of_processes_for_frontend_testing = 20
 number_of_processes_for_batch_testing    = 20
-#age_reuse_previous_test=600
+#age_reuse_previous_test                 = 600
```

Comment:
The `number_of_processes_*` keys have also default values in config.pm. I suggest them to be commented too:
```
# number_of_processes_for_frontend_testing = 20
# number_of_processes_for_batch_testing    = 20
```